### DISCUSSION TITLE
setDragImage not working

### DISCUSSION THREAD

**Discussion by magnus-trent** [xyflow/xyflow#2770] [Q&A]
### Describe the Bug

Referencing the comment on [React.js - setDragImage on draggable element](https://stackoverflow.com/questions/45052198/react-js-setdragimage-on-draggable-element), I can't seem to get the drag image to actually work. I've tried a variety of implementations but none actually work. Only one time on accident did the preview change to show the nodes on the flow somehow.

```tsx
import vec from "~/public/next.svg";
import img from "~/public/test.png"
import fav from "~/public/favicon.ico";
...
// TODO: Currently broken, fix later.
  useEffect(() => {
    const image = new Image(20);
    image.src =
      img.src;

    image.onload = () => {
      console.log("loaded");
      setDragImage(image);
    };

    // setDragImage(image);
  }, [setDragImage]);

  /**
   * When a node is dragged, update the graph.
   */
  const onDragOver = useCallback(
    (event: DragEvent<HTMLDivElement>) => {
      event.preventDefault();

      event.dataTransfer.dropEffect = "move";
      event.dataTransfer.setDragImage(dragImage, 0, 0);
    },
    [dragImage]
  );
```

Does anyone know how to get this working? Could it be Next.js not letting it work? When loading the page, and logging to the console, the image and link both work, but not the drag image

### Steps to Reproduce the Bug or Issue

Code provided

### Expected behavior

The image should be used

### Platform

- OS: [e.g. macOS, Windows, Linux]
- Browser: [e.g. Chrome, Safari, Firefox]
- Version: [e.g. 91.1]

**Comment by moklick** [comment]
Could you provide a codesandbox? Not sure if this is a React Flow issue or a general issue with your implementation.

**Comment by magnus-trent** [comment]
Sorry for the late reply, https://codesandbox.io/p/github/Blackfall-Labs/visualizr/draft/throbbing-pond?file=%2Fsrc%2Fapp%2Fpage.tsx
